jcommune-service/src/test/java/org/jtalks/jcommune/service/nontransactional/BBCodeServiceTest.java

```diff
@@ -41,6 +48,15 @@ public void testQuote() {
         assertEquals(result, "[quote=\"name\"]source[/quote]");
     }
 
+    @Test
+    public void testUserbbProcessorReturnCorrectLink() {
```

Comment:
I think you mean nested `[user]` tags here? I'd update the test name to be more precise.